terraform/variables.tf

```diff
@@ -0,0 +1,40 @@
+variable aws_region {
+  default = "eu-west-1"
+}
+
+variable project_name {
+  default = "payment-reconciler"
+}
+
+variable handler {
+  default = "handler.handler"
```

Comment:
In go this should be the name of your executable. Your executable will live inside a zip. If that executable is just called `main` then this handler should be called `main`